lessons/Early-Math/associative-property/langs/en-us/pages/discover-assoc.adoc

```diff
@@ -0,0 +1,66 @@
+= Discover the Associative Property
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Take a look at the two Circles of Evaluation below. Now, evaluate them to help you decide if the Associative Property holds for problems involving *addition*.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(+ (+ 10 8) 3))} | @show{(coe  '(+ 10 (+ 8 3)))}
+| @ifnotsoln{@math{(10 + 8) +3 = ?}} @ifsoln{@math{(10 + 8) + 3 = 21}} | @ifnotsoln{@math{10 + (8 + 3) = ?}} @ifsoln{@math{10 + (8 + 3) = 21}}
```

Comment:
Still recommend using ANS? instead of question marks so it's clear that students should actually evaluate the expression

Reply:
?ANS doesn't work inside mathjax.
This is the best option there is, sadly.